courses/free-courses-pt_BR.md

```diff
@@ -162,6 +162,12 @@
 * [Curso completo e atual de HTML5 e CSS3 - Módulo 2 de 5](https://www.youtube.com/playlist?list=PLHz_AreHm4dlUpEXkY1AyVLQGcpSgVF8s) - Gustavo Guanabara (Curso em Vídeo)
 * [Curso completo e atual de HTML5 e CSS3 - Módulo 3 de 5](https://www.youtube.com/playlist?list=PLHz_AreHm4dmcAviDwiGgHbeEJToxbOpZ) - Gustavo Guanabara (Curso em Vídeo)
 * [Curso completo e atual de HTML5 e CSS3 - Módulo 4 de 5](https://www.youtube.com/playlist?list=PLHz_AreHm4dkcVCk2Bn_fdVQ81Fkrh6WT) - Gustavo Guanabara (Curso em Vídeo)
+* [Curso de HTML, CSS e Bootstrap - Aula 1 de 6 ](https://www.youtube.com/watch?v=QZ8mu8vuOpg&list=PL8iIphQOyG-Cv3auRYoZtbvzJJrsvdMy-&index=1) - Michelli Brito (YouTube)
```

Comment:
is there no playlist we can use for this?
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
Maybe we should find a different link for the Gustavo Guanabara modules.